daemons/dss-scalability-test/domovoilib/dynamodb/__init__.py

```diff
@@ -0,0 +1,83 @@
+import boto3
+
+dynamodb_client = boto3.client('dynamodb')
+
+SCALABILITY_TEST_TABLE = 'scalability_test'
+SCALABILITY_TEST_RUN_TABLE = 'scalability_test_run'
```

Comment:
These table names are not sufficiently differentiated. Can you rename scalability_test to something more specific (scalability_test_results?)